### ISSUE TITLE
Remove setting default spellchecker

### ISSUE THREAD

**Issue by Zodiac1978** [bueltge/de_DE.php#13]
We can safely remove this function as the spellchecker is removed since 5 years:
https://core.trac.wordpress.org/ticket/23341#comment:9

https://github.com/bueltge/de_DE.php/blob/master/de_DE.php#L273-L293

**Comment by bueltge** [comment]
@Zodiac1978 thanks for the hint. I removed that part of the file. Maybe we get also more changes with Gutenberg?

**Comment by Zodiac1978** [comment]
Yes, maybe. WP 4.9.9 will be focused also on i18n, so maybe there is a chance to get this into core?
https://make.wordpress.org/core/2018/09/12/wordpress-4-9-9-minor-release-roadmap/

Or, as an alternative, couldn't we add this file to the dist folder? So adding this just to the German locale?
https://make.wordpress.org/polyglots/handbook/translating/packaging-localized-wordpress/working-with-the-translation-repository/#dist

This file normalizes filenames for example which core is still not doing at the moment:
https://core.trac.wordpress.org/ticket/22363
https://core.trac.wordpress.org/ticket/30130

Or did you try this and it got a Wontfix?

**Comment by bueltge** [comment]
The normalize of the file names on this plugin works only for our language, not globally, like Arabian or Chinese. So that I think this is not a solution for the core. However is not really nice that this tickets are so ling open :( We should use our enhancement, the de_DE.php or other plugin in this context. That's also the point that this solution here is usable as dropin and/or plugin.

We can add this file to the Dropin list, it will only run, if the languge settings in the back end is set to German, so that the core checks for a file inside `wp-content/languages` with the language key, in this context `de_DE.php`. So yes, you can add this to the #dist list, but the solution is more optional, also useable as plugin. The run as plugin has the benefit that the install get updates from w.org - https://wordpress.org/plugins/de_de/

**Comment by bueltge** [comment]
A update on w.org is necessary, however we can close this issue @Zodiac1978 ?

**Comment by bueltge** [comment]
Thanks for your help, and correction.
